car.capnp

```diff
@@ -424,6 +424,7 @@ struct CarParams {
   safetyConfigs @62 :List(SafetyConfig);
   alternativeExperience @65 :Int16;      # panda flag for features like no disengage on gas
   maxLateralAccel @68 :Float32;
+  autoResumeSng @69 :Bool;
```

Comment:
if this is just used for docs, can you add a comment? we should be a bit better about documenting our structs